### ISSUE TITLE
Insane FPS Drop / Lag

### ISSUE THREAD

**Issue by WarpWing** [mmtrt/leagueoflegends#182]
So I got the League snap working and now League uses Vulcan. However this leaves my League install lagging. Before without using Vulcan(I was using OpenGL I believe or whatever to render it by default), I was averaging 60-70fps no problem but now Vulkan is only running at 3-5fps on all low settings. Anyway to force OpenGL while still using Vulcan for the client? I know that I had to install a vulkan winetrick to get the client to work but it makes the game unplayable. Any ideas?

**Comment by mmtrt** [comment]
open winecfg command `leagueoflegends.winecfg` then add application exe from `~/snap/leagueoflegends/common/.wine/drive_c/Riot Games/League of Legends/Game/League of Legends.exe` then set `dlloverrides` as follows
![lolcfg](https://user-images.githubusercontent.com/2322598/131226325-2e94a8e0-17f1-499d-9e6d-4cfeefeb0afb.png)

**Comment by WarpWing** [comment]
> leagueoflegends.winecfg

Sorry I'm kind of a idiot, how do I add the application exe? I'm at winecfg, idk what to click xd

**Comment by WarpWing** [comment]
![image](https://user-images.githubusercontent.com/28925758/131227747-60ed1faa-3e54-4ce4-a2d1-528f0234b89f.png)
I added 
```
~/snap/leagueoflegends/common/.wine/drive_c/Riot Games/League of Legends/Game/League of Legends.exe
```
To the new override for library box and set it to builtin and still doesn't work.

**Comment by mmtrt** [comment]
You are doing it wrong lol,

So in applications tab below there is add application button from there you have to select exe file should reside in that path then click ok then click on exe filename in Applications tab then add libraries it should show empty then its for that exe file so hit apply.

**Comment by WarpWing** [comment]
> You are doing it wrong lol,
> 
> So in applications tab below there is add application button from there you have to select exe file should reside in that path then click ok then click on exe filename in Applications tab then add libraries it should show empty then its for that exe file so hit apply.

Lmao I'm an idiot. Sorry for misreading that. Did your instructions, It works perfectly. Thanks mate!

**Comment by moxxos** [comment]
I am having the same issue with a nvidia gpu driver version: 525.60.13. Client works fine but as soon as I get into game it is 3-5 fps even on everything low in settings. I used this snap previously (I'm guessing before vulkan update) with the same gpu and it worked great. I tried adding the LoL.exe with the manual dll in winecfg as instructed here but it is not working, fps is still extremely low. Anything else I can try?
